reports/cmd/sync-vectors/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"os"
+
+	"github.com/TBD54566975/sdk-development/reports"
+	"golang.org/x/exp/slog"
+)
+
+func main() {
+	defer reports.CleanupGitAuth()
+	if err := reports.ConfigureGitAuth(); err != nil {
+		panic(err)
+	}
+
+	errs := make(map[string]error)
+	for _, sdk := range reports.SDKs {
+		if err := reports.SyncSDK(sdk); err != nil {
+			errs[sdk.Name] = err
+		}
+	}
+
+	if err := reports.CleanupGitAuth(); err != nil {
+		panic(err)
```

Comment:
why panic here and os.Exit(1) below?

Reply:
the `os.Exit(1)` below is when there are errors from processing one (or more) of the SDKs, and the structure expects errors to be associated with a particular SDK. I could rearrange it all, but realistically the only time this would go off is severe failures of ie. the underlying hardware, at which point errors from the rest of the build are probably not too relevant.